### PR TITLE
More feedback and remove OldStrMessage

### PR DESCRIPTION
**Pull request by mitchelvanamstel**
Some refactoring done and also removed OldStrMessage and made it just one which is compatible with both the new and the old wallet. Fixes starting from the wallet GUI or RPC interface

### CODE REVIEW

File: contrib/gitian-descriptors/gitian-linux.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-name: "pivx-linux-4.3"
+name: "sapphire-linux-4.3"
```
Comment (crypt0p3t3r) [inline contrib/gitian-descriptors/gitian-linux.yml]: the correct one should be `sapphire-linux-1.4`

File: contrib/gitian-descriptors/gitian-linux.yml [hunk 2]
```diff
@@ -33,7 +33,7 @@ packages:
 - "ca-certificates"
 - "python3"
 remotes:
-- "url": "https://github.com/pivx-project/pivx.git"
+- "url": "https://github.com/mitchelvanamstel/SAPP_1_4"
```
Comment (crypt0p3t3r) [inline contrib/gitian-descriptors/gitian-linux.yml]: make it `https://github.com/sappcoin-com/SAPP` the current 1_4 repository will be renamed to the main one

File: contrib/gitian-descriptors/gitian-osx.yml [hunk 3]
```diff
@@ -1,5 +1,5 @@
 ---
-name: "pivx-osx-4.3"
+name: "sapphire-osx-4.3"
```
Comment (crypt0p3t3r) [inline contrib/gitian-descriptors/gitian-osx.yml]: `sapphire-osx-1.4`

File: contrib/gitian-descriptors/gitian-osx.yml [hunk 4]
```diff
@@ -29,7 +29,7 @@ packages:
 - "python3-setuptools"
 - "fonts-tuffy"
 remotes:
-- "url": "https://github.com/pivx-project/pivx.git"
+- "url": "https://github.com/mitchelvanamstel/SAPP_1_4"
```
Comment (crypt0p3t3r) [inline contrib/gitian-descriptors/gitian-osx.yml]: `https://github.com/sappcoin-com/SAPP`

File: contrib/gitian-descriptors/gitian-win.yml [hunk 5]
```diff
@@ -1,5 +1,5 @@
 ---
-name: "pivx-win-4.3"
+name: "sapphire-win-4.3"
```
Comment (crypt0p3t3r) [inline contrib/gitian-descriptors/gitian-win.yml]: sapphire-win-1.4

File: contrib/gitian-descriptors/gitian-win.yml [hunk 6]
```diff
@@ -24,7 +24,7 @@ packages:
 - "python3"
 - "rename"
 remotes:
-- "url": "https://github.com/pivx-project/pivx.git"
+- "url": "https://github.com/mitchelvanamstel/SAPP_1_4"
```
Comment (crypt0p3t3r) [inline contrib/gitian-descriptors/gitian-win.yml]: https://github.com/sappcoin-com/SAPP

File: contrib/gitian-descriptors/gitian-linux.yml (hunk 2 above)
Comment (mitchelvanamstel) [reply contrib/gitian-descriptors/gitian-linux.yml]: Changed both values to the SAPP repo's and the version

**Comment by crypt0p3t3r** [comment]
Once again, thank you Mitchel

**Pull request merged**